test-case/check-userspace-pa-status.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+##
+## Case Name: check-userspace-pa-status
+## Preconditions:
+##    N/A
+## Description:
+##    check the pulseaudio runs or not
+## Case step:
+##    1. run pgrep pulseaudio
+##    2. check the status
+##    3. restart pulseaudio
+## Expect result:
+##    pulseaudio runs
+##
+
+# shellcheck source=case-lib/lib.sh
+source "$(dirname "${BASH_SOURCE[0]}")"/../case-lib/lib.sh
+
+func_opt_parse_option "$@"
+
+dlogi "check pulseaudio starts correctly or not"
+pid=$(pgrep -x pulseaudio)
+if [ "$?" != "0" ]; then
+    # let's try to start pulseaudio
+    pulseaudio --start
+    sleep 1
+    pid=$(pgrep -x pulseaudio)
```

Comment:
No need to search with `pgrep`, `pactl stat` finds `pulseaudio` directly, unambiguously and performs more sanity checks.